boards/arm64/xenvm/board.cmake

```diff
@@ -2,14 +2,18 @@ set(SUPPORTED_EMU_PLATFORMS qemu)
 set(QEMU_ARCH aarch64)
 
 set(QEMU_CPU_TYPE_${ARCH} cortex-a57)
-
+if(CONFIG_XENVM_USE_GIC_V3)
+    set(QEMU_GIC_VERSION 3)
+else()
+    set(QEMU_GIC_VERSION 2)
+endif()
 set(QEMU_KERNEL_OPTION -kernel ${ZEPHYR_BINARY_DIR}/xen)
 set(QEMU_FLAGS_${ARCH}
    -gdb tcp::4224
    -cpu ${QEMU_CPU_TYPE_${ARCH}} -m 6G
    -nographic
-   -machine virt,gic-version=2,virtualization=true
+   -machine virt,gic-version=${QEMU_GIC_VERSION},virtualization=true
```

Comment:
Are this changes needed if we decided to build/run qemu separately from Zephyr (build with Moulin/run with separate script)? I don`t think that Zephyr OS kernel is proper place for this. Also this will be broken with zephyr-dom0-xt, because it doesn't build Xen.

Reply:
I did this before we decided that. Other hand I had to make test. As far as I know we didn't move to run qemu separately from Zephyr